package.json

```
@@ -37,12 +37,20 @@
         "lint:fix:md": "markdownlint --fix  \"**/*.md\"",
         "lint:scss": "stylelint \"**/*.scss\"",
         "lint:fix:scss": "stylelint \"**/*.scss\" --fix",
+        "fmt": "prettier --write \"**/*.+(js|json|scss)\"",
+        "fmt:check": "prettier --check \"**/*.+(js|json|scss)\"",
```

```suggestion
        "fmt": "prettier --write .",
        "fmt:check": "prettier --check .",
```

This can now be simplified.